Author: <gaudenz@debian.org>
Description: Don't kill ourselves if not forking into the background
--- a/mouseemu.c
+++ b/mouseemu.c
@@ -706,7 +706,7 @@
 		}
 		/* tell the parent we are running without problems */
 		/* What should we do if the parent is dead? */
-		if (answer) {
+		if (answer && !nofork) {
 			answer=0;
 			kill(getppid(), SIGUSR1);
 		}
